tests/test_schema.py

```diff
@@ -63,6 +63,19 @@ def test_dump_with_strict_mode_raises_error(SchemaClass):
     assert type(exc.messages) == dict
     assert exc.messages == {'email': ['Not a valid email address.']}
 
+def test_invoke_validators_with_many():
```

Comment:
I have moved it. Let me know if it looks good or if there are other changes you would like implemented.